frontend/src/index.scss

```diff
@@ -1,13 +1,8 @@
-body {
-  margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
-}
+@import "stylebase/variables";
+@import "stylebase/type";
+@import "stylebase/utils";
+@import "stylebase/global";
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, 'Courier New',
-    monospace;
-}
+@import "components/Button/Button";
+
+@import "views/Home";
```

Comment:
Was there a reason not to import this directly in the component's *.js file? Same with Button. Wouldn't we end up with just loading the styles needed for the particular view that way – which sounds decent to me.

Reply:
I couldn't get it to work. The variables were not found in the component scss-files. Didn't want to waste more time on it. Feel free to refactor.

Reply:
Oh yea, because everything that you import into the react component is on its own. Just add an `@import "stylebase/variables"` on top. AFAIK that should fix it.